.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+## Description
+
+TODO: Include a detailed description of the changes in the body of the PR.
+
+TODO: if relevant, provide a URL to updated Pattern Library documentation.
+
+## Manual Test Plan
+
+TODO: Include instructions for any required manual tests, and any manual testing that has
+already been performed.
+
+## Testing Checklist
+- TODO: Include unit tests
+- TODO: Include accessibility (a11y) tests
+- [ ] Manually test right-to-left languages and i18n
+  of the changes.
+- [ ] Manually test responsive behavior.
+
+## Non-testing Checklist
+- TODO: Tag DevOps on your PR (user `edx/devops`), either for review
+  (more specific to devops = better) or just a general FYI.
+- TODO: Consider any documentation your change might need, and which
+  users will be affected by this change.
+- TODO: Double-check that your commit messages will make a meaningful release note.
+- TODO: Consult with the Pattern Library team if adding a new component.
+
+## Post-review
+- TODO: Squash commits into discrete sets of changes (see the note about release notes above).
+
+## Reviewers
+If you've been tagged for review, please check your corresponding box once you've given the :+1:.
+- [ ] Code review #1 (TODO: tag a specific user)
+- [ ] Code review #2 (TODO: tag a specific user)
+- [ ] Docs review (required for UI strings/error messages)
+- [ ] UX review
```

Comment:
What is meant by UX review? Review by a UX team member? If so, same comment as above. If it means manual testing of the UX by anyone, then it would require a sandbox on which we can deploy a test page with the specified component